src/commands/install.rs

```diff
@@ -27,6 +28,9 @@ pub fn install() -> Result<()> {
         GITHUB_API_REPOS_BASE_URL, FUEL_CORE_REPO, RELEASES_LATEST
     ))?;
 
+    info!("Fetching forc {}", &forc_release_latest_tag);
+    info!("Fetching fuel-core {}", &fuel_core_release_latest_tag);
```

Comment:
Should we move this to after `forc` is fetched?

Reply:
Ohhh my apologies, I thought these were meant to indicate when the fetching of each package begins, so I thought each of these should happen before their corresponding download begins.